org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/AbstractPointsModel.java

```diff
@@ -39,22 +39,23 @@ public void addPropertyChangeListener(PropertyChangeListener listener) {
 	public void removePropertyChangeListener(PropertyChangeListener listener) {
 		this.pcs.removePropertyChangeListener(listener);
 	}
-	
+
 	@FieldDescriptor(visible=false)
 	private String name;
 
+	@Override
 	public String getName() {
 		return name;
 	}
 	public void setName(String name) {
 		this.name = name;
 	}
-	
+
 	@Override
 	public List<String> getScannableNames() {
 		return Arrays.asList(getName());
 	}
-	
+
 	@UiHidden
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "UiHidden"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)